atintegrators/AperturePass.c

```diff
@@ -8,15 +8,16 @@ struct elem {
 
 void AperturePass(double *r_in, double *limits, int num_particles)
 {
-    /* Checks X and Y of each input 6-vector and marks the corresponding element in
-     * lossflag array with 0 if X,Y are exceed the limits given by limitsptr array
-     * limitsptr has 4 elements: (MinX, MaxX, MinY, MaxY)
+    /* Checks X,Y of each input 6-vector and marks the corresponding element in
+     * lossflag array with 0 if X,Y are exceed the limits given by limits and limitslong
+     * arrays
+     * limits has 4 elements: (MinX, MaxX, MinY, MaxY)

```

Comment:
@lfarv I think this change is correct no? Why `limitsptr`? For the rest it is cosmetic, I have seen such changes in other PR in many occasions...

Reply:
What is `limitslong` ? But I agree with the change from `limitsptr` to `limits`.

Reply:
I forgot to remove 'limitslong' from my previous try to handle both the longitudinal and transverse apertures in a single integrator. I corrected the docstrings in both integrators. There was also a mention of 'limitslong' in LongtAperturePass.c because I copy passed the docstring.